### DISCUSSION TITLE
Increase time until containers are marked as unhealthy by e.g. 60 sec

### DISCUSSION THREAD

**Discussion by frederikb96** [nextcloud/all-in-one#3197] [Ideas] [answered]
I am using Nextcloud AIO on a vserver with 8 vcores and 32 GB Ram. When starting the containers, the `nextcloud-aio-nextcloud` and the `nextcloud-aio-notify-push` containers are always unhealthy for ~40 seconds.

Related to issue #1611 and [this command](https://github.com/nextcloud/all-in-one/issues/1611#issuecomment-1366641549), but back then it was a slightly different problem and that got fixed :)

My config for Nextcloud AIO is quite basic, only thing I changed: `- NEXTCLOUD_ADDITIONAL_APKS=imagemagick ffmpeg perl`

If I understand it correctly the containers take too long for the startup and thus they are unhealthy for some time until the startup phase is over and everything is running. Since I am using a server, which imho is normally powerful and I am not installing lots of other packages when starting the containers, I would think that others also encounter this "problem".

Consequently, I would like to ask if it is possible to increase the time until the `nextcloud-aio-nextcloud` and the `nextcloud-aio-notify-push` containers are marked as unhealthy by e.g. 60 sec. This would be beneficial for monitoring tools, which report back if containers are unhealthy.

Thanks :)

**Comment by szaimen** [comment, marked as answer]
Will be resolved in https://github.com/nextcloud/all-in-one/pull/3207 and https://github.com/nextcloud/all-in-one/pull/3208
